### ISSUE TITLE
switching database crashes app

### ISSUE THREAD

**Issue by ricoderks** [ndcn/omicser#80]
Switching database crashes the app with error:

```
[1] "in observeEvent(input$SI_database, "
Warning: Error in setnames: Can't assign 4 names to a 1 column data.table
  [No stack trace available]
[1] "input$SI_subset no yet updated"
[1] "db type - Lipidomics-omics"
```

Switching from Domenico to Yassene dataset.

**Comment by ricoderks** [comment]
@ergonyc I mainly focussed on where the issue is. As far as I can see it takes too long for `rv_selections` to be updated.

**Comment by ricoderks** [comment]
This one can be closed. Fixed by @ergonyc in PR #84.
